pkgs/operator/operator.go

```diff
@@ -132,10 +134,14 @@ func New(key *rsa.PrivateKey) *Server {
 }
 
 func (s *Server) Start(port uint16) error {
-	s.Logger.Infof("server listening for incoming requests on port %d", port)
 	srv := &http.Server{Addr: fmt.Sprintf(":%v", port), Handler: s.Router}
```

Comment:
Not part of this PR, but it'd be nice if we can add timeouts to the `http.Server`

Reply:
We have Chi limiting requests to routes, to prevent DDOS, if I understood you correctly